mocha.el

```diff
@@ -85,10 +96,12 @@ If MOCHA-FILE is specified run just that file otherwise run
 MOCHA-PROJECT-TEST-DIRECTORY.
 
 IF TEST is specified run mocha with a grep for just that test."
-  (let ((path (or mocha-file mocha-project-test-directory))
+  (let* ((path (or mocha-file mocha-project-test-directory))
         (target (if test (concat "--grep \"" test "\" ") ""))
         (node-command (concat mocha-which-node (if debug (concat " --debug=" mocha-debug-port) "")))
         (options (concat mocha-options (if debug " -t 21600000"))))
```

Comment:
If you're setting the other special option below, maybe this should be pulled out two and both of those should go into another function?